proto/raft_serverpb.proto

```diff
@@ -29,6 +29,8 @@ message RaftMessage {
     bytes extra_ctx = 11;
 
     disk_usage.DiskUsage disk_usage = 12;
+    // Used to measure the send wait duration.
+    uint64 last_observed_time = 13;
```

Comment:
As `BatchRaftMessage` has one, why bother recording it here?

Reply:
Looks like we don't need to add a new field in protobuf, as step 1 occurs within the process with no inter-network communication.